presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/RowExpressionFormatter.java

```diff
@@ -57,7 +64,13 @@ public String formatRowExpression(RowExpression expression)
         @Override
         public String visitCall(CallExpression node, Void context)
         {
-            return node.getDisplayName() + "(" + String.join(", ", formatRowExpressions(node.getArguments())) + ")";
+            if (standardFunctionResolution.isArithmeticFunction(node.getFunctionHandle()) || standardFunctionResolution.isComparisonFunction(node.getFunctionHandle())) {
```

Comment:
Could you also handle the `CAST` case?
For `CAST(a)` we can print out as `CAST(a AS <return_type>)`. This helps us to understand the type info.

Also, can we handle `NEGATION` and `SUBSCRIPT` as well?

Reply:
Sure thing, should I also include `AND` and `OR`.

Reply:
sounds good!